src/TestFramework/TestFramework/Assertions/Assert.AreEqual.cs

```diff
@@ -1190,9 +1170,7 @@ public static void AreNotEqual(double notExpected, double actual, double delta,
     /// Thrown if <paramref name="expected"/> is not equal to <paramref name="actual"/>.
     /// </exception>
     public static void AreEqual(string? expected, string? actual, bool ignoreCase)
```

Comment:
I missed these `string` APIs where I could also put the `NotNullIfNotNull` attribute.